network-api/manage.py

```diff
@@ -4,7 +4,7 @@
 
 # Django runs twice to support live-reloading, so check Django's internal settings to determine whether or not
 # to start the debugger.
-if (os.environ.get("RUN_MAIN") or os.environ.get("WERKZEUG_RUN_MAIN")) and os.environ.get("VSCODE_DEBUGGER"):
+if (os.environ.get("RUN_MAIN") or os.environ.get("WERKZEUG_RUN_MAIN")) and os.environ.get("VSCODE_DEBUGGER") == "True":
```

Comment:
It was not, in the sense that there was no way to disable the debugger setup. Environment variables are strings, so setting `VSCODE_DEBUGGER=False` in `.env` would still pass this check, because the string `"False"` is still a "truthy" value.

So now, you will need to make sure to set `VSCODE_DEBUGGER=True` in `.env` to activate the debugger setup (which seems like it was the intended functionality of this check. 

I need to be able to disable the debugger because I don't use VSCode and therefore need a way to connect a different debugger. With the debugger server running though, the breakpoints would be swallowed by that and my debugger would never start.